conf/configuration.go

```diff
@@ -113,7 +113,7 @@ type MailerConfiguration struct {
 }
 
 type PhoneProviderConfiguration struct {
-	Disabled bool `json:"disabled"`
+	Enabled bool `json:"enabled"`
```

Comment:
should we explicitly set `default:"false"` here?

Reply:
by default, it's automatically set to false. But yeah i should include that for readability.